src/rabbit_mgmt_stats_gc.erl

```diff
@@ -66,7 +66,7 @@ start_link(Table) ->
 
 init([Table]) ->
     {ok, Interval} = application:get_env(rabbit, collect_statistics_interval),
-    rabbit_log:info("Statistics garbage collector started for table ~p.~n", [{Table, Interval}]),
+    rabbit_log:info("Statistics garbage collector started for table ~p at interval ~p.~n", [Table, Interval]),
```

Comment:
This makes it sound like the sample GC process is started periodically while the interval defines how often GC "runs" are performed. What do you think about re-wording it a bit like "Statistics garbage collector started for table ~p with run  interval ~p."?